modules/installation-mirror-repository.adoc

```diff
@@ -225,7 +225,7 @@ mirrored, extract it and pin it to the release:
 +
 [source,terminal]
 ----
-$ oc adm release extract -a ${LOCAL_SECRET_JSON} --icsp-file=<file> \ --command=openshift-install "${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${OCP_RELEASE}"
+$ oc adm release extract -a ${LOCAL_SECRET_JSON} --icsp-file=<file> --command=openshift-install "${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${OCP_RELEASE}-${ARCHITECTURE}"
```

Comment:
this file is not related to monitoring, but checked with oc client 4.16.0-0.nightly-2024-07-25-145501, there is not `icsp-file` parameter, need the image registry QE to check
```
# oc version
Client Version: 4.16.0-0.nightly-2024-07-25-145501
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3

# oc adm release extract --help | grep -i icsp-file
no result
```

Reply:
this was mistakenly included in the PR. Will remove it.